src/adapters/hasura/altStudent.adapter.ts

```diff
@@ -1041,4 +1050,138 @@ export class ALTStudentService {
       data: responseData,
     });
   }
+  public async updateStudent(userId, request, body) {
```

Comment:
@YoginiTayade This API should only be accessible to the admin users, I guess admin check is missing

Reply:
when the request is made, in hasura and keyclock it checks for permissions and if the permission is allowed it lets it process but if not it returns an error 
should I add an extra response to return that "the user does not have permission"?

Reply:
resolve